dags/check_miovision.py

```diff
@@ -1,3 +1,8 @@
+"""
```

Comment:
can we change this dag's name to be `miovision_check`?

Reply:
WAIT there is another miovision_check!??!

Reply:
I figured since the new `miovision_check` has none of the same tasks as existing `check_miovision`, it would be best to duplicate and rename rather than update existing.